res/css/views/elements/_GenericEventListSummary.scss

```diff
@@ -16,6 +16,75 @@ limitations under the License.
 
 .mx_GenericEventListSummary {
     position: relative;
+
+    &:not([data-layout=bubble]) {
```

Comment:
I'm inclined to change this into:

````
&[data-layout=group],
&[data-layout=irc]
````

in order to prevent unexpected regressions from happening.

Reply:
Since no review is ongoing, I'm implementing that.